python/run_benchmark.sh

```diff
@@ -362,10 +362,10 @@ fi
 # Logistic Regression Classification
 if [[ "${MODE}" =~ "logistic_regression" ]] || [[ "${MODE}" == "all" ]]; then
     if [[ ! -d ${gen_data_root}/classification/r${num_rows}_c${num_cols}_float32.parquet ]]; then
+    # same data set as random forest classifier
```

Comment:
assuming this is intentional?

Reply:
I think so. LR benchmark can reuse the classification dataset of random forest classifier.